templates/standard.adoc

```diff
@@ -0,0 +1,34 @@
+= Pomegranates
+Author Minetest-Docs and Contributors 
+v1.0, 2022-01-01
+
+:description: A sample template for using AsciiDoc.
+:keywords: sample
+// TODO: figure out why there is no table of contents!
+:toc:
+
+// Using an image directory is a good idea, for relative paths!
+:imagesdir: ../assets
+
+// URL references:
+:url-source: https://github.com/minetest/minetest_docs
+
+This is a sample template of how a good AsciiDoc can be structured.
+This pragraph describes the purpose of the document.
+In this sample, an imaginary API
+for producing pomegranates will be documented.
+
+So without further ado, here is what we should see when we are done.
+
+[#img-pomegranate]
+.The Expected Result
+image::pomegranate.png[Pomegranate]
+
+== Pomegranate API Methods
+
+* `plant_tree()`
```

Comment:
```suggestion
=== `plant_tree(pos, name)`
* `pos`: Vector
* `name`: string

Plants a tree of type `name` at `pos` if `name` is a registered pomegranate tree.

.Planting A Tree
[example, lua]
---
plant_tree(Vector.new(35, 42, 36), "regular")
---

```

Reply:
Not happy with the `Vector`. Should be lowercase and monospaced.

Reply:
Agreed.

Reply:
`Vector` is a class name in this examples (like it is in MT), but I agree with it being monospaced.

Reply:
Can you add the monospace? And then I will commit the suggestions.

Reply:
I already made `Vector` monospaced, also make `string` monospaced.